### PR TITLE
Add June General Meeting minutes

### PR DESCRIPTION
**Pull request by rnjudge**

### CODE REVIEW

**Comment by rnjudge** [comment]
Hold on merging this until group can provide approval at July 13th General Meeting.

File: general/2023-06-01.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+* Sandeep Patil (Philips)
+* Rose Judge (VMware)
+* Steve Winslow
+* Jilayne Lovejoy (Red Hat)
+* Brad Goldring (GTC Law Group)
+* David Edelsohn
+* Holger Frydrych
+* Jordi Mon
+* Jeff Buddington
+* Jeff Mendoza (Kusari)
```
Comment (jeff-schutt) [inline general/2023-06-01.md:100]:
```suggestion
* Jeff Mendoza (Kusari)
* Jeff Schutt (Cisco)
```

**Comment by rnjudge** [comment]
Joe Bussell (Microsoft) was present.

**Comment by rnjudge** [comment]
These minutes were approved at the General Meeting call today. Merging.

**Pull request merged**